src/components/drops/mixins/index.d.ts

```diff
@@ -26,5 +26,5 @@ export interface AlignProps {
 }
 
 export interface StretchProps {
-  streach: boolean | "width" | "align"
+  streach?: boolean | "width" | "align"
```

Comment:
typo

Reply:
Nice catch @novykh !
I will open a separate issue for this one, since I need to check it's usage before fixing the typo!

Reply:
Yes, thought it was passed as `streach`, but checked it as well, so I am pushing an update on that! :)